libs/localization/src/lib/en.json

```diff
@@ -1296,6 +1296,9 @@
       "columnFour": "Predicted Y",
       "columnFive": "Other metadata",
       "filter": "Filter",
+      "panelExplanation": "Explanation",
+      "panelInformation": "Information",
+      "panelTitle": "Selected Instance",
```

Comment:
nit: I think we try to keep things sentence case, so "Selected instance" is preferred over "Selected Instance"